stock_quant_manual_assign/models/stock_picking.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Mikel Arregi - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.multi
+    def action_assign(self):
+        self.mapped('pack_operation_ids').unlink()
```

Comment:
This one is not needed on odoo/OCB vanilla code, isn't it? Anyway, I think is harmless, but please make sure, and comment why is here.